azure-iot-device/azure/iot/device/common/pipeline/pipeline_ops_base.py

```diff
@@ -129,7 +143,7 @@ class DisableFeatureOperation(PipelineOperation):
     Even though this is an base operation, it will most likely be handled by a more specific stage (such as an IoTHub or MQTT stage).
     """
 
-    def __init__(self, feature_name, callback=None):
+    def __init__(self, feature_name, callback):
```

Comment:
>callback [](start = 37, length = 8)

Good call. Optional callbacks make testing a pain, and there's no reason for it. This is a pattern I introduced that I regret.